### PR TITLE
Create new note 

### LINKED ISSUE #10: create new note ui

### PR DESCRIPTION
**Pull request by nancy-binani**

### CODE REVIEW

**Comment by yedhink** [comment]
@nancy-binani I will review this PR once you go through https://handbook.neetokb.com/articles/git-and-github-essentials and fix the issues.

I believe this is the [second time](https://github.com/nancy-binani/wheel/pull/8#issuecomment-1252388392) I am pointing this out. Lemme know if something is not clear in the article.

**Comment by nancy-binani** [comment]
I am really sorry this mistake won't happen again.

File: app/javascript/src/components/Dashboard/Notes/List/index.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from "react";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/List/index.jsx]: The List folder only has one file. Which is suggesting to us that we don't need a folder for it in the fist place. Thus keep this logic within `Notes/List.jsx`

File: app/javascript/src/components/Dashboard/Notes/List/index.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          <div className="flex flex-row gap-2">
+            <Clock />
+            <Typography style="body2">
+              {`${note.status} ${calculateCreatedAgo(note.createdAt)}`}
+            </Typography>
+            <Avatar
+              size="small"
+              user={{
+                imageUrl: "https://i.pravatar.cc/300",
+                name: "Nancy",
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/List/index.jsx]: Go through the Slack thread. I have pointed out about personas to be use.

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 3]
```diff
@@ -3,11 +3,73 @@ import * as yup from "yup";
...
+  {
+    label: "User flow",
+    value: "value5",
+  },
+  {
+    label: "V2",
+    value: "value6",
+  },
+];
+
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js]: Go through the Slack thread. We can build these values by creating a util called something like `buildSelectOptions`.

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 4]
```diff
@@ -3,11 +3,73 @@ import * as yup from "yup";
...
+    .required("Please select a contact"),
+  tags: yup
+    .array()
+    .of(
+      yup.object().shape({
+        label: yup.string().oneOf(TAGS.map(tag => tag.label)),
+        value: yup.string().oneOf(TAGS.map(tag => tag.value)),
+      })
+    )
+    .min(1, "Please select a tag")
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js]: Here the error message can be made more specific. Here we should be pointing out that atleast one tag ought to be selected. That suggests two things:
- The least amount of tags needed
- More than one tag can also be selected

**Review by yedhink — CHANGES_REQUESTED** [review]
@nancy-binani

**Comment by nancy-binani** [comment]
I have added the required changes.Please review them.

**Comment by yedhink** [comment]
@nancy-binani Already provided the reviews: https://github.com/nancy-binani/wheel/pull/13#pullrequestreview-1116921138

**Comment by nancy-binani** [comment]
I meant I have done all the changes. So please review them.

**Comment by nancy-binani** [comment]
This message was like my final message which informs all the review changes are done.So you can review them altogether.

File: app/javascript/src/components/Dashboard/Notes/List.jsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Typography, Dropdown, Button, Avatar } from "neetoui";
+
+import { MENU_OPTIONS as menuOptions } from "./constants";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/List.jsx]: https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react#:~:text=Use%20upper%20snake%20case%20in%20constant%20name

**Review by yedhink — CHANGES_REQUESTED** [review]
@nancy-binani

**Comment by nancy-binani** [comment]
I have updated according to naming conventions.Please review it.

**Pull request merged**